### PR TITLE
Set up CI for Python 3.

### PR DESCRIPTION
**Pull request by oliverchang**

### CODE REVIEW

File: src/local/butler/integration_tests.py [hunk 1]
```diff
@@ -28,14 +29,21 @@
 
 def execute(_):
   """Run integration tests."""
+  if sys.version_info.major == 3:
+    command = 'run_server3'
+    indicator = b'Booting worker'
```
Comment (inferno-chromium) [inline src/local/butler/integration_tests.py]: Why is the indicator different here ?

Comment (oliverchang) [reply src/local/butler/integration_tests.py]: We're no longer using dev_appserver, but a different server framework.

**Pull request merged**